tests/context/conftest.py

```diff
@@ -34,22 +34,46 @@ def ambulation_negative_data_50hz(path_tests):
 
 
 @fixture(scope="class")
-def motion_positive_data(path_tests):
+def motion_positive_data_100hz(path_tests):
+    fs = 100
     accel = read_csv(
         path_tests / "context" / "data" / "motion_test_positive.csv"
     ).values
+    accel = vstack([accel, accel, accel])
+    time = arange(0, len(accel) / fs, 1 / fs)
 
-    fs = 100.0
-
-    return accel, fs
+    return time, accel, fs
 
 
 @fixture(scope="class")
-def motion_negative_data(path_tests):
+def motion_negative_data_100hz(path_tests):
+    fs = 100
     accel = read_csv(
         path_tests / "context" / "data" / "motion_test_negative.csv"
     ).values
+    time = arange(0, len(accel) / fs, 1 / fs)
+
+    return time, accel, fs
+
+
+@fixture(scope="class")
+def motion_positive_data_20hz(path_tests):
+    fs = 20
+    accel = read_csv(
+        path_tests / "context" / "data" / "motion_test_positive.csv"
+    ).values
+    accel = vstack([accel, accel, accel])
+    time = arange(0, len(accel) / fs, 1 / fs)
+
+    return time, accel, fs
 
-    fs = 100.0
 
-    return accel, fs
+@fixture(scope="class")
+def motion_negative_data_20hz(path_tests):
+    fs = 20
+    accel = read_csv(
+        path_tests / "context" / "data" / "motion_test_negative.csv"
```

Comment:
can negative motion test data just be created instead of having to maintain a file?

Reply:
Leaving as file, as discussed.